client/client.go

```diff
@@ -148,6 +149,8 @@ type clientConfig struct {
 	failoverGracePeriod time.Duration
 	// watcher is a constructor function that creates a new Watcher
 	watcher WatcherCtor
+	// autoWatch causes the client to being watching immediately so that new rasndomness is proactively fetched.
```

Comment:
```suggestion
	// autoWatch causes the client to start watching immediately in the background so that new randomness is proactively fetched and added to the cache.
```